test/executions/evaluation_test.exs

```diff
@@ -626,6 +626,45 @@ defmodule Wanda.Executions.EvaluationTest do
                result: :warning
              } = Evaluation.execute(execution_id, group_id, checks, gathered_facts, %{})
     end
+
+    test "should return a critical result agent times out and severity is warning" do
```

Comment:
Maybe there's a missing `when` in the sentence
> should return a critical result _when_ agent times out and severity is warning

Reply:
Will change to `...critical result if an agent times ...` to follow the same syntax than the others.